src/types/present.ts

```diff
@@ -0,0 +1,15 @@
+export type PresentationFrame<T> = T extends Array<unknown>
```

Comment:
Maybe there is a difference but can we not to `PresentationFrame = DisclosureFrame`

Reply:
Maybe the disclosure frame can extend the presentation frame and add the decoy count on every level.

Reply:
I'll try to create a generic type, where you can pass extra properties.

However I noticed some issues with this type in general, that if you use the default record<string, unknown> it will not accept all types correctly

Reply:
Okay combined them 👍